docs/en/query_language/create.md

```diff
@@ -107,7 +107,8 @@ It is not possible to set default values for elements in nested data structures.
 
 ### Constraints {#constraints}
 
-WARNING: This feature is experimental. Correct work is not guaranteed on non-MergeTree family engines.
+!!!warning "Warning"
```

Comment:
Not experimental anymore. It's production quality feature. It is supported for all kind of tables.